app/Http/Requests/StorePost.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StorePost extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|max:191',
+            'entity_id' => 'required',
```

Comment:
Worth adding a `|numeric|exists:entities,id` to make sure it's a valid entity_id before saving to the database. Also worth validating `visibility_id` the same way it's validated in other rules.